### ISSUE TITLE
Expose wallet creation methods

### ISSUE THREAD

**Issue by aesedepece** [witnet/sheikah#28]

**Comment by aesedepece** [comment]
```typescript
API.Mnemonics.generate(): Promise<string>

API.Seed.deriveFromMnemonics(mnemonics: string): Promise<Seed>

API.Seed.deriveFromEntropy(entropy: Buffer): Promise<Seed>

API.PrivateKey.deriveChild(parentKey: PrivateKey, childIndex: number): Promise<PrivateKey>

API.PrivateKey.deriveDescendant(masterKey: PrivateKey, keyPath: KeyPath): Promise<PrivateKey>
```

**Comment by aesedepece** [comment]
My previous comment no longer applies as per the latest arguments discussed in #49.

**Comment by aesedepece** [comment]
This shall be implemented as handler functions (see #27)

**Comment by aesedepece** [comment]
Replaced by #217
